cmd/describe/machinepool/cmd_test.go

```diff
@@ -176,7 +176,7 @@ var _ = Describe("Upgrade machine pool", func() {
 				stdout, stderr, err := test.RunWithOutputCaptureAndArgv(runWithRuntime, testRuntime.RosaRuntime,
 					Cmd, &[]string{nodePoolName})
 				Expect(err).ToNot(BeNil())
-				Expect(err.Error()).To(ContainSubstring(fmt.Sprintf("Machine pool '%s' not found", nodePoolName)))
+				Expect(err.Error()).To(ContainSubstring(fmt.Sprintf("Node pool '%s' not found", nodePoolName)))
```

Comment:
@hunterkepley I don't think we should change to `Node pool`. `Machine pool` is the customer visible term and also what's used to execute this i.e `rosa describe machinepool`.

Reply:
understood